script/Ciel/Day 3/03_01_CIEL03_3_3.txt

```diff
@@ -359,7 +359,7 @@ I reflexively jerk backwards.
 [sha:904fbc83be5118d790f42ec6ac002cd1ea2d8fa7]{
 -- Page 182, Offset 5318.
 -- どこまで本気なのか、笑顔で先輩は出かけてしまった。
-Senpai leaves with a smile on her face while I'm still trying to work out how serious she is.
+Senpai leaves with a smile on her face while I'm still trying to work out how serious she was.
```

Comment:
On the fence about this one. The statement she made is in the past, but she's presumably still embodying whatever seriousness she had in that moment right now (basely any time has passed), so I might keep it ongoing

Reply:
I prefer "is"